quic-client/src/lib.rs

```diff
@@ -204,7 +205,7 @@ impl ConnectionManager for QuicConnectionManager {
     }
 
     fn new_connection_config(&self) -> QuicConfig {
```

Comment:
I suppose there are probably backward compatibility concerns, but the name of this method doesn't make a lot of sense now.